### PR TITLE
Improved macos guide

### PR DESCRIPTION
**Pull request by defnax**

### CODE REVIEW

**Comment by defnax** [comment]
@csoler

**Comment by defnax** [comment]
@csoler

File: build_scripts/OSX/MacOS_X_InstallGuide.md [hunk 1]
```diff
@@ -104,11 +119,11 @@ In QtCreator Projects -> Build -> Build Settings -> Build Steps -> Add Additiona
 
 ## Set your Mac OS SDK version
 
-Edit RetroShare.pro  
+Edit RetroShare.pro and set your installed sdk version example for 11.1 -> rs_macos11.1
 
     CONFIG += c++14 rs_macos11.1
 
-and then retroshare.pri
+Edit retroshare.pri
```
Comment (csoler) [inline build_scripts/OSX/MacOS_X_InstallGuide.md]: Why does the user need to edit retroshare.pri? There should be a section for MacOS that does the work, right?

Comment (defnax) [reply build_scripts/OSX/MacOS_X_InstallGuide.md:126]: you need look latest changes
**Edit RetroShare.pro and set your installed sdk version example for 11.1 -> rs_macos11.1**
https://github.com/RetroShare/RetroShare/pull/2797/commits/31552b111914446d040faceb15c51d80c7488218

Comment (defnax) [reply build_scripts/OSX/MacOS_X_InstallGuide.md]: its hardcoded that thing, we need one time edit to set the sdk version, seems not working from commandline to set
https://decovar.dev/blog/2021/04/01/macos-building-retroshare/

Comment (defnax) [reply build_scripts/OSX/MacOS_X_InstallGuide.md]: image
image

Comment (defnax) [reply build_scripts/OSX/MacOS_X_InstallGuide.md]: > Why does the user need to edit retroshare.pri? There should be a section for MacOS that does the work, right?

i has a newer version for edit
`Edit retroshare.pri and set your installed sdk version example for 11.1 -> rs_macos11.1 ( line 129:)`

**Comment by defnax** [comment]
@csoler

**Comment by defnax** [comment]
please merge

**Pull request merged**